src/sqlfluff/core/rules/config_info.py

```diff
@@ -42,6 +42,10 @@
         "validation": ["consistent", "qualified", "unqualified"],
         "definition": "The expectation for references in single-table select.",
     },
+    "fix_inconsistent_to": {
+        "validation": [None, "qualified", "unqualified"],
+        "definition": "Fixing behaviour for inconsistent single-table select.",
+    },
```

Comment:
To be honest I'm not sure we should add the extra config.

If people have a preference they should set it in `single_table_references` and if they don't, then we can just pick first one and go with that.

The only thing this gives you is option to save `None` but then that's confusing in itself. Does that mean don't fix? Or is it only used when `single_table_references` is not set to `consistent`?

Reply:
So I'm sort of with you. adding config shoudn't be done without thought.
My main things is this - **Every rule that is fixable should be fixable**

This is a really nice property of SQLFluff and linters in general (if you think of the workflows involving precommit etc).

The "consistent" setting I could argue against, but in so far as it exists it is still fixable - we just need to know to which state.

Seems a bit weird (as just picking "qualified" would also work), however if you have a huge code base (like I do) and you have a handful of in consistent files seems like we should fix them if we could. (Right now I have to either change everything or... hand change things).
The argument for having consistent at (which I guess is "well thats how my code base is now") is the same one for allowing us to fix those files that fail to be consistent

Reply:
Yeah I'm not against fixing `consistent`. I just think we should pick one, rather than offer another option to the user. If they really don't care (and picking `consistent` seems to suggest they don't) then there's no "wrong" answer. So pick one!

I originally thought picking based on first option makes most sense. so this:

```sql
SELECT col1, a.col2 FROM table1 a
```

Should be fixed to:

```sql
SELECT col1, col2 FROM table1 a
```

Defaulting to `qualified` would also work though and if that's easier than I say go for it. As I say IMHO there is no wrong answer here.